apps/battery/app.js

```diff
@@ -51,7 +51,7 @@ module.exports = function (activity) {
 
   function powerStatusChange (isOnline, isPlaying, testPercent) {
     logger.log('powerStatusChanged ', isOnline, isPlaying, testPercent)
-    activity.light.play('self://battery.js', { isAcConnect: isOnline })
+    activity.light.play('system://power.js', { plug: isOnline })
```

Comment:
Why do this light as system? If yes, we should provide this at framework, too.

Reply:
> Why do this light as system? If yes, we should provide this at framework, too.

If the light not a system effect, we can't change its priority. In addition, I think this effect should be system sound.

Reply:
What's its priority definition? If that must be a prioritized system sound, I also think we should think about adding to framework by default. Or move to custom-able product repo.

Reply:
> What's its priority definition? If that must be a prioritized system sound, I also think we should think about adding to framework by default. Or move to custom-able product repo.
This effect should be the highest priority, it will be played in any case. However, I think the light priority mechanism may need to be redesigned.